syncstorage-postgres/src/db/mod.rs

```diff
@@ -25,6 +27,12 @@ pub use batch_impl::validate_batch_id;
 
 const TOMBSTONE: i32 = 0;
 
+lazy_static! {
+    static ref PRETOUCH_NDT: NaiveDateTime = chrono::DateTime::parse_from_rfc3339(PRETOUCH_TS)
+        .expect("PRETOUCH_TS must be a valid RFC3339 datetime string")
+        .naive_utc();
+}
+
```

Comment:
Nit: apparently we can construct this as a `const` with all this (this is the DateTime version you'll want w/ TIMESTAMPTZ).
```suggestion
const PRETOUCH_DT: DateTime<Utc> = chrono::NaiveDate::from_ymd_opt(1, 1, 1)
        .unwrap()
        .and_hms_opt(0, 0, 0)
        .unwrap()
        .and_utc();
```
seems worth it to avoid lazy_static/its needed dereferencing. Unfortunately we can't spit out `to_rfc3339` as a const, we could just leave it as a string as is in the spanner code which is all it needs anyway.